cs3/storage/provider/v1beta1/provider_api.proto

```diff
@@ -724,3 +726,18 @@ message CreateHomeResponse {
   // Opaque information.
   cs3.types.v1beta1.Opaque opaque = 2;
 }
+
+message DeleteHomeRequest {
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 1;
```

Comment:
Having `DeleteHomeRequest` as optional leaks implementation details, for instance, we can call `DeleteHome` call with no arguments and this **COULD** translate into "delete home for the authenticated user", having a side effect.

I'm just saying that we have a chance to make this a pure function and have no side effects by adding a [user](https://cs3org.github.io/cs3apis/#cs3.identity.user.v1beta1.User) to the request 🤷

Reply:
I would be fine for that. The reason I did it like that is that I wanted the API to be similar to CreateHome.

Reply:
I think CreateHome should also use a user parameter.